docs/config.adoc

```diff
@@ -309,6 +309,10 @@ Please contribute to the document if you are able!
 The number used for a custom key represents the converted value for an OsCode in
 base 10. This differs between Windows-hooks, Windows-interception, and Linux.
 
+Running katana with the `--debug` flag lets you read the correct number number, show in parenthesis of `code` in the `KeyEvent` log lines.
```

Comment:
```suggestion
Running kanata with the `--debug` flag lets you read the correct number,
shown in parenthesis of `code` in the `KeyEvent` log lines.
```

Reply:
Fix typo, extra word, and change to use sembr

Reply:
Thanks! Excuse my Github illiteracy, but how should I best apply your suggestion? I'm thinking that a local amend and force push will make the cleanest history, if that works well with the review process. (I don't see any amend button in Github 🤔 )

Reply:
I squash all PRs anyway so whatever works for you is fine.